Queue/QueueImp.java

```diff
@@ -0,0 +1,55 @@
+// This program is to implement a queue.
+// Given a list of numbers in a queue and removed the head of queue.
+
+
+package Queue;
+
+import java.util.LinkedList;
+import java.util.Queue;
```

Comment:
Although We highly encourage you to implement without built-in libraries, This Looks Good to me!

@deephunt3r Take a look and merge it!